command/alloc_restart.go

```diff
@@ -31,6 +31,9 @@ General Options:
 
 Restart Specific Options:
 
+  -task <task-name>
+	Specify the individual task to restart.
```

Comment:
The help text above refers to the `<task>` argument, so it'd probably be a good idea to include the note you have below (in the website docs) about what happens if you provide both.

(Same for `alloc signal`)